src/components/ProjectVersion/index.jsx

```diff
@@ -26,4 +185,13 @@ const ContentsWrapper = styled.div`
   padding: 64px;
 `;
 
+const HorizontalAlign = styled.form`
+  display: flex;
+
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: flex-end;
+  width: 100%;
```

Comment:
P1: `display` 속성끼리는 같은 속성이라 같이 붙여주시는 게 좋을 것 같습니다! :)

```suggestion
  width: 100%;
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: flex-end;
```